requirements.txt

```diff
@@ -9,3 +9,7 @@ nose-timer
 # necessary for accessing SecretStorage keyring (system wide Gnome keyring)
 # and nice to have under tox for that
 # dbus-python
+
+# -- required for building docs
+sphinx
+sphinx_rtd_theme
```

Comment:
btw -- working on fixing up for it on datalad side atm and then will propagate all those changes here.